main/User_config.h

```diff
@@ -74,11 +74,24 @@ const byte subnet[] = {255, 255, 255, 0};
 
 #if defined(ESP8266) || defined(ESP32) // for nodemcu, weemos and esp8266
 //#  define ESPWifiManualSetup true //uncomment you don't want to use wifimanager for your credential settings on ESP
+//#  define MQTT_HTTPS_FW_UPDATE //uncomment to enable updating via mqtt message.
 #else // for arduino boards
 const byte ip[] = {192, 168, 1, 99};
 const byte mac[] = {0xDE, 0xED, 0xBA, 0xFE, 0x54, 0x95}; //W5100 ethernet shield mac adress
 #endif
 
+#ifdef MQTT_HTTPS_FW_UPDATE
+//If used, this should be set to the root CA certificate of the server hosting the firmware.
+// The certificate must be in PEM ascii format
+const char* https_fw_server_cert = CERT_ATTRIBUTE = R"EOF("
```

Comment:
I think CERT_ATTRIBUTE  macro should be defined before if MQTT_HTTPS_FW_UPDATE || SECURE_CONNECTION are defined

Reply:
Good timing, was fixing this right now (pushed before all commits). Will be updated in a minute.

Reply:
While I'm at it, maybe that macro should be removed? Esp32 and 8266 code compile fine with just PROGMEM being there.

Reply:
ok!

Reply:
Done, macro removed and I just pushed the changes.